src/rp2_common/pico_bootsel_via_double_reset/pico_bootsel_via_double_reset.c

```diff
@@ -14,12 +14,12 @@
 #define PICO_BOOTSEL_VIA_DOUBLE_RESET_TIMEOUT_MS 200
 #endif
 
-// PICO_CONFIG: PICO_BOOTSEL_VIA_DOUBLE_RESET_ACTIVITY_LED, GPIO to use as bootloader activity LED when BOOTSEL mode is entered via reset double tap (or -1 for none), type=int, default=-1, group=pico_bootsel_via_double_reset
+// PICO_CONFIG: PICO_BOOTSEL_VIA_DOUBLE_RESET_ACTIVITY_LED, GPIO to use as bootloader activity LED when BOOTSEL mode is entered via reset double tap (or -1 for none), type=int, max=29, default=-1, group=pico_bootsel_via_double_reset
```

Comment:
i think we should actually switch to using undefined rather than -1 now to match regular LED